### PR TITLE
Add options for enabledTests and enabledSpecifications

### PR DESCRIPTION
**Pull request by Fishbowler**

### CODE REVIEW

**Comment by Fishbowler** [comment]
Uh-oh. I need to look into that failure.

File: src/jobs/test.yml [hunk 1]
```diff
@@ -47,4 +55,6 @@ steps:
       adminAccountPassword: << parameters.adminAccountPassword >>
       disabledSpecifications: << parameters.disabledSpecifications >>
       disabledTests: << parameters.disabledTests >>
+      enabledSpecifications: << <parameters.enabledSpecifications >>
```
Comment (guusdk) [inline src/jobs/test.yml:58]: There's one to many `<` in this line.

File: src/jobs/test.yml [hunk 2]
```diff
@@ -47,4 +55,6 @@ steps:
...
+      enabledTests: << <parameters.enabledTests >>
```
Comment (guusdk) [inline src/jobs/test.yml:59]: There's also a redundant `<` character on this line.

**Comment by Fishbowler** [comment]
Thanks Guus :heart:

**Review by guusdk — APPROVED** [review]

**Pull request merged**